rules/windows/process_creation/win_apt_apt29_thinktanks.yml

```diff
@@ -17,7 +17,7 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine: '*-noni -ep bypass $*'
+        CommandLine|contains: '-noni -ep bypass $'
```

Comment:
better to avoid spaces in such scenarios (: could you please change it to the:

```
        CommandLine|contains|all: 
            - '-noni'
            - '-ep'
            - 'bypass'
            - '$'
```

Reply:
Fixed :)